pkg/controller/plan/adapter/ovirt/client.go

```diff
@@ -49,6 +50,19 @@ func (r *Client) CreateSnapshot(vmRef ref.Ref) (snapshot string, err error) {
 		return
 	}
 	snapsService := vmService.SnapshotsService()
+
+	locked, err := r.isAnySnapshotLocked(snapsService)
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	if locked {
+		err = web.ProviderNotReadyError{
+			Provider: r.Source.Provider,
+		}
+		return
+	}
```

Comment:
where is the 3-times limit that is mentioned in the description of the PR?

Reply:
not 3-times but every 3s, and that's the default for the reconcile 
https://github.com/kubev2v/forklift/blob/42043cca14acbf58268eb78020722b795c853da4/pkg/controller/plan/migration.go#L33

Reply:
ah, three seconds - ok, that makes more sense :)